model-api/bpmn-model/pom.xml

```diff
@@ -13,7 +13,13 @@
 
   <name>camunda BPM - BPMN Model API</name>
   <inceptionYear>2014</inceptionYear>
-
+  <properties>
+    <camunda.artifact>org.camunda.bpm.model.bpmn</camunda.artifact>
+    <camunda.osgi.export>
+      !org.camunda.bpm.model.bpmn.schema,
```

Comment:
This module is the only one where this is defined as an exclusion although all model-related modules contain such a package. Could you go into more detail why this is needed here?

Reply:
This package only consists XSD files, and there is no Java classes. I don't think it's necessary to export this package.

Reply:
Thanks for that input! I think you are correct and we don't need to export this. As all other model-related modules contain the same package, do you think we should exclude them there as well?

Reply:
Would you mind to point which package that you refer to?

Reply:
If I am not mistaken, the other model modules (CMMN and DMN) also contain such schema files at `org.camunda.bpm.model.cmmn.schema` and `org.camunda.bpm.model.dmn.schema` respectively.